### ISSUE TITLE
Can we Convert Ifc to Svg File With XBim Toolkit OR In any otherway

### ISSUE THREAD

**Issue by umarone** [xBimTeam/XbimWebUI#417]
Hi Everyone,
                    I want to ask you people can we convert our ifc file to svg format and can get data from svg image on click events, if so please guide. 
Moreoever if some paid solutions are available then please also you can guide me... 

Thankx

**Comment by andyward** [comment]
Yes, we have a service as part of xbim Flex that does this. See: https://developers.xbim.net/reference/floorplans_getfloorplan - i.e. once an IFC model is in Flex we can generate smart SVG floor plans for each Storey.

I believe we may be speaking in the new year, but let me know in advance if there's a wider use case.

**Comment by umarone** [comment]
Sir If you allow me , I want to share a video with you , so you can see my requirements.. what you say please

**Comment by andyward** [comment]
Feel free to email me - but note we're shut for Christmas holidays so any
response may be delayed,

Andy

**Comment by umarone** [comment]
Sir what is your email please

**Comment by andyward** [comment]
Should be on my profile: https://github.com/andyward
